### PR TITLE
Tweak perf test code and default settings

### PR DESCRIPTION
**Pull request by artem-v**

### CODE REVIEW

File: reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java [hunk 1]
```diff
@@ -40,30 +41,45 @@
...
+            .doOnSuccess(System.out::println)
             .block();
 
     Disposable report = startReport();
 
-    Supplier<Payload> payloadSupplier = () -> ByteBufPayload.create(BUFFER.retainedSlice());
-
-    Flux.range(1, (int) Configurations.NUMBER_OF_MESSAGES)
+    Mono.fromCallable(() -> ByteBufPayload.create(BUFFER.retainedSlice()))
+        .repeat(Configurations.NUMBER_OF_MESSAGES)
```
Comment (bsideup) [inline reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java]: `Mono.fromCallable` + `.repeat` is much slower than `Flux.range(0, (int) Configurations.NUMBER_OF_MESSAGES`

File: reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java [hunk 2]
```diff
@@ -35,31 +35,48 @@
...
+                      System.out.println("streaming " + msgNum + " messages ...");
 
-                        return Mono.fromCallable(payloadCallable)
-                            .subscribeOn(Schedulers.parallel())
-                            .repeat(Configurations.NUMBER_OF_MESSAGES);
-                      }
-                    }))
+                      return Flux.range(1, msgNum)
+                          .map(i -> ByteBufPayload.create(BUFFER.retainedSlice()))
+                          .subscribeOn(Schedulers.parallel());
```
Comment (bsideup) [inline reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: FYI I am investigating why it doesn't work without `subscribeOn`, will post the result here soon-ish

File: reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java (hunk 1 above)
Comment (artem-v) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java]: Ok
I will revert to   FLux.range

File: reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java (hunk 2 above)
Comment (artem-v) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: In the first place the issue is with memory leak ;  this server tps  thing doesn't last even 3 minutes

Comment (bsideup) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: there is no "leak", but the server for some reason does not send the payload to the client (yet drains the publisher)

/cc @OlegDokuka

Comment (artem-v) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: But, indeed, tps is much bigger with this code, but memory leak is killing it all.

File: reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java (hunk 1 above)
Comment (artem-v) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyPing.java]: Reverted to Flux range

**Comment by artem-v** [comment]
I encourage to use the scripts: 
```
reactor-aeron-tools/scripts/netty
reactor-aeron-tools/scripts/rsocket/netty
```

File: reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java (hunk 2 above)
Comment (artem-v) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: > there is no "leak"

it's not that leak which you are supposed to find with `ResourceLeakDetector`, but if go and look at jconsole you will see that heap grows and grows and grows

Comment (ronenhamias) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: maybe lets open an issue with this info ? so we can merge this one and re-run tests on aws?

Comment (artem-v) [reply reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java]: https://github.com/rsocket/rsocket-java/issues/514#issuecomment-467872520

**Comment by artem-v** [comment]
Seen this issue at https://github.com/rsocket/rsocket-java/issues/514#issuecomment-467872520

**Pull request merged**